package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "theming",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
version should be changed when releasing, because you know only then what version it will be